### PR TITLE
Midnight time bug

### LINKED ISSUE #87: Events not properly registering accross midnight
image

It should be wed 10pm to  thursday 5am

`[USER COMMAND][add hi from 10pm to 5am]`

### PR DESCRIPTION
**Pull request by burnflare**
Fixes #87

### CODE REVIEW

File: src/main/java/seedu/agendum/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -99,6 +101,14 @@ public AddCommand(String name, Optional<LocalDateTime> endDateTime, String perio
         );
     }
 
+    public Optional<LocalDateTime> incrementEndTimeIfNecessary(Optional<LocalDateTime> startDateTime, Optional<LocalDateTime> endDateTime) {
+        if (startDateTime.get().compareTo(endDateTime.get()) >= 1) {
+            return Optional.of(endDateTime.get().plusDays(1));
```
Comment (burnflare) [inline src/main/java/seedu/agendum/logic/commands/AddCommand.java:106]: Er, quite a stupid way to do it, I know.

Comment (burnflare) [reply src/main/java/seedu/agendum/logic/commands/AddCommand.java:106]: @rachx got a better way?

**Comment by INCENDE** [comment]
@burnflare does this fix #78 also?

**Comment by burnflare** [comment]
Oops this needs to be opened again, against master

**Pull request merged**